### DISCUSSION TITLE
Change Microsoft Office App Themes

### DISCUSSION THREAD

**Discussion by Robert13guo** [t1m0thyj/WinDynamicDesktop#307] [Ideas]
I used this for a long time and it is fantastic!
But could we have a script or function to change the Microsoft Office App Themes?

**Comment by t1m0thyj** [comment]
If anyone finds where Office stores its configuration for light/dark theme, feel free to contribute a PowerShell script that toggles the setting to the [WDD-scripts](https://github.com/t1m0thyj/WDD-scripts) repo.
